modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/visible/fields/VisibleFieldsDDMFormFieldType.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.form.field.type.internal.visible.fields;
+
+import com.liferay.dynamic.data.mapping.form.field.type.DDMFormFieldType;
+import com.liferay.dynamic.data.mapping.form.field.type.constants.DDMFormFieldTypeConstants;
+import com.liferay.dynamic.data.mapping.form.field.type.internal.select.SelectDDMFormFieldType;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Rodrigo Paulino
+ */
+@Component(
+	immediate = true,
+	property = {
+		"ddm.form.field.type.name=" + DDMFormFieldTypeConstants.VISIBLE_FIELDS,
+		"ddm.form.field.type.system=true"
+	},
+	service = DDMFormFieldType.class
+)
+public class VisibleFieldsDDMFormFieldType extends SelectDDMFormFieldType {
```

Comment:
Hey @rodrigopaulino ,

As we discussed, It would be nice to rename this class and things related replacing VisibleFields* to MultiLanguageOptionSelect*

Thanks!

Reply:
Done https://github.com/liferay-data-engine/liferay-portal/pull/515/commits/46bd4c78d79bf55086b32f0522b842cffd7ade17